src/coffee/import.coffee

```diff
@@ -396,6 +396,24 @@ class Import
       i += chunkSize
     return chunkifiedActionsArray
 
+  updateProductInBatches: (existingProduct, chunkifiedUpdateRequests) ->
+    # Send update requests in series and for every request set the latest prod.version
+    # see: http://bluebirdjs.com/docs/api/promise.reduce.html
+    Promise.reduce(chunkifiedUpdateRequests, (existingProduct, updateRequest) =>
+      updateRequest.version = existingProduct.version
+      productsUri = createRequestBuilder({ @projectKey })
+        .products
+        .byId(existingProduct.id)
+        .build()
+      request = {
+        uri: productsUri
+        method: 'POST'
+        body: updateRequest
+      }
+      @client.execute request
```

Comment:
it's been a while I've read ☕️
This is supposedly returns the promise of the request, right?

Reply:
yes - it will return a promise. And in the coffeescript the value from the last line in the function is returned from the function